.travis.yml

```diff
@@ -4,7 +4,10 @@ jdk:
   - oraclejdk8
   - openjdk7
 
-
+  
+before_install:
+        - sudo apt-get install libboost-dev libboost-test-dev
```

Comment:
Of course, did not notice that was in the yaml file.. Good as it is :)

On Wed, Jan 21, 2015, 21:17 Waleri Enns <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/wenns/sonar-cxx/pull/387#discussion_r23325522>:
>
> > @@ -4,7 +4,10 @@ jdk:
> >    - oraclejdk8
> >    - openjdk7
> >
> > -
> > +
> > +before_install:
> > +        - sudo apt-get install libboost-dev libboost-test-dev
>
> The .travis.yml will be executed on a Travis VM, which is garanteed to be
> a debian derivate ;)
> Having said this, the line is not actually needed, as the behave will only
> run 'sonar-runner' against checked in reports, no?
> The boost header, libs, etc. are needed only for developing the tests,
> i.e. on your workstation. Not on the CI slave.
>
> So not probs here
>
> wont be executed on Windows
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wenns/sonar-cxx/pull/387/files#r23325522>.
>